hugo/layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,33 @@
+{{- $id := .Get "id" | default (.Get 0) -}}
+{{- $title := .Get "title" | default "" }}
+{{- $image := .Get "image" | default "" }}
+{{- $url := "https://www.youtube.com/watch?v={{ $id }}" }}
+{{- $contentClass := "video__content" }}
+{{- $lazy := cond (or (eq (.Get "lazy") false) (eq (.Get "lazy") "false")) false true -}}
+
+{{ if not $title }}
+    {{- $contentClass = print $contentClass " video__content--no-title" -}}
+{{ end }}
+
+<div class="video" data-youtube="{{ $id }}">
+    {{ if $image }}
+        <img class="video__image" src="{{ $image }}" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
+    {{ else }}
+        <img class="video__image" src="https://img.youtube.com/vi/{{ $id }}/maxresdefault.jpg" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
+    {{ end }}
+
+    <a href="{{ $url }}" target="_blank" class="video__button">
+        <div class="{{ $contentClass }}">
+            {{ if $title }}
+                <h2 class="video__title">{{ $title }}</h2>
+            {{ end }}
+
+            <svg xmlns="http://www.w3.org/2000/svg" class="video__icon" viewBox="0 0 24 24">
```

Comment:
Is there a reason we're not using the icon spritesheet here?

Reply:
This icon needs to fill colors, the icon spritesheet can only handle one